Makefile

```diff
@@ -44,8 +45,7 @@ check_version:
 	./$(BINARY_NAME) version
 
 setup:
-	$(GOCMD) get -u github.com/golang/dep/cmd/dep
-	$(GODEP) ensure
+	go mod download
```

Comment:
I think we should add a `go mod tidy` somewhere

Reply:
Done!